docs/developer-docs/integrations/bitcoin/ckbtc.md

```diff
@@ -2,37 +2,476 @@
 
 ## Overview
 
-Chain-key Bitcoin (ckBTC) is an [ICRC-1](https://github.com/dfinity/ICRC-1/blob/aa82e52aaa74cc7c5f6a141e30b708bf42ede1e3/standards/ICRC-1/README.md)-compliant token that
-is backed 1:1 by Bitcoin held 100% on the IC mainnet.
+Chain-key Bitcoin (ckBTC) is an [ICRC-1](https://github.com/dfinity/ICRC-1/blob/aa82e52aaa74cc7c5f6a141e30b708bf42ede1e3/standards/ICRC-1/README.md)-compliant token that is backed 1:1 by Bitcoin held 100% on the IC mainnet.
 
 The ckBTC functionality is provided through an interplay of two canisters:
 - The **ckBTC minter**. 
 - The **ckBTC ledger**.
 
-The ckBTC minter is responsible for the minting and burning of ckBTC tokens.
-A certain amount of tokens is minted if a user transfers the same amount of bitcoin to a
-specific Bitcoin address under the ckBTC minter's control. The Bitcoin address uniquely
-identifies the owner of the sent bitcoins, making it possible for the ckBTC minter to
-associate the minted ckBTC funds with the correct owner.
-Conversely, an owner of ckBTC can request a retrieval of bitcoin to a Bitcoin address of his or
-her choosing.
-The ckBTC tokens are burned and the corresponding BTC amount is transferred using a regular
-Bitcoin transaction.
-A detailed description of the ckBTC minter can be found in its [GitHub repository](https://github.com/dfinity/ic/tree/master/rs/bitcoin/ckbtc/minter).
+The ckBTC ledger is responsible for keeping account balances and for transferring ckBTC between accounts. It provides the following functionality:
+- It enables the ckBTC minter to mint and burn ckBTC.
+- It enables the transfer of ckBTC among users.
+
+The ckBTC minter is responsible for the minting and burning of ckBTC tokens. A certain amount of tokens is minted if a user transfers the same amount of Bitcoin to a specific Bitcoin address under the ckBTC minter's control. The Bitcoin address uniquely identifies the owner of the sent Bitcoins, making it possible for the ckBTC minter to associate the minted ckBTC funds with the correct owner. The ckBTC minter needs to receive BTC, based on a large number of confirmations due to the lack of finality in Bitcoin, before it mints the same amount in ckBTC. The ckBTC minter burns the ckBTC before it transfers the corresponding BTC amount is transferred using a regular Bitcoin transaction.
```

Comment:
```suggestion
The ckBTC minter is responsible for the minting and burning of ckBTC tokens. A certain amount of tokens is minted if a user transfers the same amount of bitcoin to a specific Bitcoin address under the ckBTC minter's control. The Bitcoin address uniquely identifies the owner of the sent bitcoins, making it possible for the ckBTC minter to associate the minted ckBTC funds with the correct owner. The ckBTC minter needs to receive BTC, based on a large number of confirmations due to the lack of finality in Bitcoin, before it mints the same amount in ckBTC. The ckBTC minter burns the ckBTC before it transfers the corresponding BTC amount using a regular Bitcoin transaction.
```

Reply:
Note that "Bitcoin" refers to the technology (e.g., "Bitcoin network", "Bitcoin address", ...), whereas "bitcoin" refers to the currency (e.g., "I received 1 bitcoin").